test/framework/envs/kubernetes/env.go

```diff
@@ -97,3 +97,8 @@ func PrintKubeState(report ginkgo.Report) {
 		}
 	}
 }
+
+func ExpectCpToNotCrash() {
+	restartCount := framework.RestartCount(Cluster.GetKuma().(*framework.K8sControlPlane).GetKumaCPPods())
+	Expect(restartCount).To(Equal(0))
```

Comment:
```suggestion
	Expect(restartCount).To(Equal(0), "CP restarted in this suite, this should not happen.")
```

This will make the output easier to understand than: 
```
  [FAILED] Expected
      <int>: 3
  to equal
      <int>: 0
```